extensions/mapstore/src/test/java/com/hazelcast/mapstore/GenericMapLoaderTest.java

```diff
@@ -75,6 +82,8 @@ public static void beforeClass() {
     public void setUp() {
         hz = instances()[0];
         mapName = "people_" + randomName();
+        assertThat(databaseProvider).isInstanceOf(GenericMapStoreObjectProvider.class);
+        objectProvider = (GenericMapStoreObjectProvider) databaseProvider;
```

Comment:
So now that we have the `GenericMapStoreObjectProvider` could we have another parent class - not `JdbcSqlTestSupport`? (which is specific to Jdbc, but we use something else for Mongo).

Reply:
Done :) Multiple revisions, so just please take a look overall